app/models/dialog_field_sorted_item.rb

```diff
@@ -10,6 +10,16 @@ def initialize_with_values(dialog_values)
     end
   end
 
+  def force_multi_value
+    return true if options[:force_multi_value].present? &&
+                   options[:force_multi_value] != "null" &&
+                   options[:force_multi_value]
+  end
+
+  def force_multi_value=(setting)
+    options[:force_multi_value] = setting
+  end
+
```

Comment:
I don't think it makes sense to put these two methods on `dialog_field_sorted_item`, because the drop down is the only one that actually uses it. Radio buttons by definition of the element can only have one item selected. I could see a scenario where "radio buttons" get turned into a list of checkboxes or something when "force multi value" is enabled, but I would argue that's a new feature and doesn't belong in this PR.

If this was put in in response to @lpichler's comment, I would simply suggest adding the code you put in `#determine_selected_default_value` into `dialog_field_drop_down_list.rb` and call super afterwards, like this:

```ruby
def determine_selected_default_value
  # new code
  super
end
``` 

Thoughts?

Reply:
@eclarizio `dialog_field_tag_control` also supports multi-values and inherits from this model.

Adding 
```ruby
def force_multi_value
  false
end
```

into radio button's model as we discussed.